waiter/src/waiter/util/utils.clj

```diff
@@ -251,6 +252,10 @@
   (let [error-context (build-error-context data-map request)
         content-type (request->content-type request)]
     {:body (case content-type
+             "application/grpc"
```

Comment:
What do gRPC clients generally do with the body content when `grpc-status` indicates an error?

Reply:
I get a `gRPC frame header malformed: reserved bits not zero` message. I believe it has to do with the grpc encoding of the body.

https://github.com/grpc/grpc-java/blob/d7b9438d399e44d9dccd0f7830cb544bb1840f0d/core/src/main/java/io/grpc/internal/MessageDeframer.java#L372

Even if the header was set, I think this is problematic because grpc will try to parse the body as the declared response type from the proto file.

Reply:
Should we avoid providing a body then?  Should we look at what other proxies do?

Reply:
Will fix.

Reply:
Avoided body.

Envoy produces responses with approrpiate grpc status headers, example:

https://github.com/envoyproxy/envoy/blob/fd7c70b3f3739554974769f1d0e72b706eb5c5a3/source/extensions/filters/http/grpc_http1_reverse_bridge/filter.cc#L154-L158

https://github.com/envoyproxy/envoy/blob/15706964a29e595c045a5d7ef0646d04f347dcc1/source/common/grpc/status.cc#L6-L26